frontend/src/metabase-types/api/parameters.ts

```diff
@@ -46,7 +46,7 @@ export interface Parameter extends ParameterConfig {
   value?: any;
 }
 
-export interface ParameterConfig {
+export interface ValuesConfig {
```

Comment:
Should we maybe call it `ParameterValuesConfig`? It can be confusing when you import `ValuesConfig` from `metabase-types/api`